Sell/Basket/Models/Basket.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Basket.swift
+//  RealifeTech
+//
+//  Created by Mickey Lee on 23/06/2021.
+//  Copyright © 2021 Realife Tech. All rights reserved.
+//
+
+import Foundation
+
+public struct Basket {
+
+    public let grossAmount: Int?
+    public let discountAmount: Int?
+    public let netAmount: Int?
+    public let seatInfo: [SeatInfo]?
```

Comment:
Could you let me know why we don't expose [String: String?] ?

- [ ] Please let me know

Reply:
FE agrees to have `seatInfo: SeatInfo` exposed on the SDK. Besides, BE has updated this field from `seatInfo: [SeatInfo]` to:
```
seatInfo: SeatInfo

type SeatInfo {
Row: String
Seat: String
Block: String
Table: String
}
```
Note. BE will fix the property names to camel-cased and I will pull the latest schema again after BE resolves the names.